superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -65,8 +67,25 @@ def _init_distributed_setting(self):
                         ' distributed implementation: {}.'.format(self._name, self._args.distributed_impl)
                     )
                     return False
-
-                torch.distributed.init_process_group(backend=self._args.distributed_backend.value)
+                try:
```

Comment:
It looks like that we prefer this solution. Why not do `try: torch.distributed.init_process_group(backend=self._args.distributed_backend.value)`

Reply:
Yes, that's why we should `try` it first.